frontend-public/src/customPropTypes/permits.js

```diff
@@ -0,0 +1,18 @@
+import { PropTypes, shape } from "prop-types";
+
```

Comment:
Do Proponents see permits?

Reply:
No but i needed this to make the mine custom prop type work.

Reply:
Removed.